### DISCUSSION TITLE
How to properly disable warnings and set logging to error for every library mkdocs uses?

### DISCUSSION THREAD

**Discussion by l1x** [mkdocs/mkdocs#3581] [Q&A]
Our mkdocs -q gh-deploy --force command produces a millions of lines of warning despite the fact mkdocs was invoked with -q.

Is there a way to disable this behaviour for good?

```bash
source venv/bin/activate
  mkdocs -q gh-deploy --force
  shell: /usr/bin/bash -e {0}
  
WARNING:root:
                [git-revision-date-localized-plugin] Running on GitHub Actions might lead to wrong
                Git revision dates due to a shallow git fetch depth.

                Try setting fetch-depth to 0 in your GitHub Action
                (see https://github.com/actions/checkout).
                
WARNING:root:
                [git-revision-date-localized-plugin] Running on GitHub Actions might lead to wrong
                Git revision dates due to a shallow git fetch depth.

                Try setting fetch-depth to 0 in your GitHub Action
                (see https://github.com/actions/checkout).
```

**Comment by oprypin** [comment]
I think this is an issue with git-revision-date-localized-plugin. (And I've personally observed similar with mkdocs-macros-plugin.)

The particular plugins should be able to be configured to not produce such loud messages.

MkDocs works as intended by silencing only INFO level messages, not WARNING.

Although worth noting that in your case it does seem like the plugin's warning is very true and relevant - it seems like the plugin isn't doing anything unless you resolve the warning like it says. Otherwise you may as well just remove the plugin.
